docs/source/conceptual/tensor_parallelism.md

```diff
@@ -0,0 +1,15 @@
+# Tensor Parallelism
+
+Tensor parallelism (also called horizontal model parallelism) is a technique used to fit a large model in multiple GPUs.  Intermediate outputs between ranks are sent and received from one rank to another in a synchronous or asynchronous manner. When multiplying input with weights for inference, multiplying input with weights directly is equivalent to dividing the weight matrix column-wise, multiplying each column with input separately, and then concatenating the separate outputs like below 👇 
```

Comment:
I think that the wording could improve a bit
* is the synchronous or asynchronous part important for explaining TP?
* you say the intermediate outputs. I'm not sure this is very clear
* TP distributes chunks of layers across multiple GPUs. This is not mentioned/clear in the definition

Reply:
I addressed rest of your comments.